snafu/utils/get_prometheus_data.py

```diff
@@ -13,23 +12,29 @@
 class get_prometheus_data():
     def __init__(self, action):
 
+        self.sample_info_dict = action
         self.uuid = action["uuid"]
         self.user = action["user"]
         self.cluster_name = action["cluster_name"]
         self.test_config = action["test_config"]
 
         # change datetime in seconds string to datetime object
-        starttime = datetime.fromtimestamp(int(action["starttime"]))
-        self.start = starttime.datetime()
+        starttime = datetime.fromtimestamp(int(self.sample_info_dict["starttime"]))
+        self.start = starttime
 
         # change datetime in seconds string to datetime object
-        endtime = datetime.fromtimestamp(int(action["endtime"]))
+        endtime = datetime.fromtimestamp(int(self.sample_info_dict["endtime"]))
         # add 120s buffer to end time
-        endtime = endtime + timedelta(seconds=120)
-        self.end = endtime.datetime
+        # endtime = endtime + timedelta(seconds=120)
```

Comment:
It looks like we're just setting self.end to datetime.fromtimestamp(int(self.sample_info_dict["endtime"])). can we simplify line 26-29 to
```
self.end = datetime.fromtimestamp(int(self.sample_info_dict["endtime"]))
```
?

Reply:
corrected, but this was old.